make generate-types creates uncompilable code

If I run `make generate-types` , I can no longer build the cassandra-operator.

```
make setup
cd cassandra-operator
go build cmd/operator/main.go

make generate-types

go build cmd/operator/main.go 
# github.com/sky-uk/cassandra-operator/cassandra-operator/pkg/client/clientset/versioned/scheme
pkg/client/clientset/versioned/scheme/register.go:39:2: undefined: "github.com/sky-uk/cassandra-operator/cassandra-operator/vendor/k8s.io/apimachinery/pkg/util/runtime".Must
```

--- a/cassandra-operator/Gopkg.lock
+++ b/cassandra-operator/Gopkg.lock
@@ -53,12 +53,12 @@
   version = "v1.1.1"
 
 [[projects]]
-  digest = "1:2cd7915ab26ede7d95b8749e6b1f933f1c6d5398030684e6505940a10f31cfda"
-  name = "github.com/ghodss/yaml"
+  digest = "1:2fd7ea1c64915c77ef4649b1b2113f449897d00721383925eca9a1c4b4071910"
+  name = "github.com/evanphx/json-patch"
   packages = ["."]
   pruneopts = "T"
-  revision = "0ca9ea5df5451ffdf184b4428c902747c2c11cd7"
-  version = "v1.0.0"
+  revision = "5858425f75500d40c52783dce87d085a483ce135"
+  version = "v4.2.0"
 
 [[projects]]
   digest = "1:f5ccd717b5f093cbabc51ee2e7a5979b92f17d217f9031d6d64f337101c408e4"
@@ -131,14 +131,6 @@
   pruneopts = "T"
   revision = "20f1fb78b0740ba8c3cb143a61e86ba5c8669768"
   version = "v0.5.0"
-
-[[projects]]
-  branch = "master"
-  digest = "1:0778dc7fce1b4669a8bfa7ae506ec1f595b6ab0f8989c1c0d22a8ca1144e9972"
-  name = "github.com/howeyc/gopass"
-  packages = ["."]
-  pruneopts = "T"
-  revision = "bf9dde6d0d2c004a008c27aaee91170c786f6db8"
 
 [[projects]]
   digest = "1:8f20c8dd713564fa97299fbcb77d729c6de9c33f3222812a76e6ecfaef80fd61"
@@ -216,6 +208,13 @@
   packages = [
     ".",
     "config",
+    "ginkgo",
+    "ginkgo/convert",
+    "ginkgo/interrupthandler",
+    "ginkgo/nodot",
+    "ginkgo/testrunner",
+    "ginkgo/testsuite",
+    "ginkgo/watch",
     "internal/codelocation",
     "internal/containernode",
     "internal/failer",
@@ -437,6 +436,26 @@
   revision = "85acf8d2951cb2a3bde7632f9ff273ef0379bcbd"
 
 [[projects]]
+  branch = "master"
+  digest = "1:bd1c9a65c8f6ca0536e5493f9769a68a583ac347f1b08d57eead592b2e17e9f9"
+  name = "golang.org/x/tools"
+  packages = [
+    "go/ast/astutil",
+    "go/gcexportdata",
+    "go/internal/gcimporter",
+    "go/internal/packagesdriver",
+    "go/packages",
+    "go/types/typeutil",
+    "imports",
+    "internal/fastwalk",
+    "internal/gopathwalk",
+    "internal/module",
+    "internal/semver",
+  ]
+  pruneopts = "T"
+  revision = "2d16b83fe98cd1bed9e2ce9fdc86bd438d14aab7"
+
+[[projects]]
   digest = "1:1469235a5a8e192cfe6a99c4804b883a02f0ff96a693cd1660515a3a3b94d5ac"
   name = "google.golang.org/appengine"
   packages = [
@@ -495,43 +514,51 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:512a6fa04ac2c195d2ebea80c7052f737d02912525268def76e9111a7ce9e6eb"
+  digest = "1:1b8ef8497e747560c6c95e3df5977e0a387b3babf8613272d97d37b3c539818d"
   name = "k8s.io/api"
   packages = [
-    "admissionregistration/v1alpha1",
     "admissionregistration/v1beta1",
     "apps/v1",
     "apps/v1beta1",
     "apps/v1beta2",
+    "auditregistration/v1alpha1",
     "authentication/v1",
     "authentication/v1beta1",
     "authorization/v1",
     "authorization/v1beta1",
     "autoscaling/v1",
     "autoscaling/v2beta1",
+    "autoscaling/v2beta2",
     "batch/v1",
     "batch/v1beta1",
     "batch/v2alpha1",
     "certificates/v1beta1",
+    "coordination/v1",
+    "coordination/v1beta1",
     "core/v1",
     "events/v1beta1",
     "extensions/v1beta1",
     "networking/v1",
+    "networking/v1beta1",
+    "node/v1alpha1",
+    "node/v1beta1",
     "policy/v1beta1",
     "rbac/v1",
     "rbac/v1alpha1",
     "rbac/v1beta1",
+    "scheduling/v1",
     "scheduling/v1alpha1",
+    "scheduling/v1beta1",
     "settings/v1alpha1",
     "storage/v1",
     "storage/v1alpha1",
     "storage/v1beta1",
   ]
   pruneopts = "T"
-  revision = "173ce66c1e39d1d0f56e0b3347ff2988068aecd0"
+  revision = "61630f889b3c3efb8c3a4ba377da1a421d9ff6ce"
 
 [[projects]]
-  digest = "1:20004d2e845a3254a057fb4ad403969bf0ab84af627027876e93a5c9ac68f3e5"
+  digest = "1:e0ec21060953ced38018fae667796890cd67dac80f969eec1634ff5ecfcf59a8"
   name = "k8s.io/apimachinery"
   packages = [
     "pkg/api/errors",
@@ -563,6 +590,7 @@
     "pkg/util/intstr",
     "pkg/util/json",
     "pkg/util/mergepatch",
+    "pkg/util/naming",
     "pkg/util/net",
     "pkg/util/runtime",
     "pkg/util/sets",
@@ -577,46 +605,56 @@
     "third_party/forked/golang/reflect",
   ]
   pruneopts = "T"
-  revision = "800fcb029b488923f14852306f0c98c11d0f360c"
+  revision = "6a84e37a896db9780c75367af8d2ed2bb944022e"
+  version = "kubernetes-1.14.1"
 
 [[projects]]
-  digest = "1:f91c1da7cb2893594908715697eb5c862f52b430f5a4be9e6d5fa9ed68b4ffaf"
+  digest = "1:651af38d5f9602fa4703f3bb3c3fdf9d74b54010cb38992ac7f9c4673eab720c"
   name = "k8s.io/client-go"
   packages = [
     "discovery",
     "discovery/fake",
     "kubernetes",
     "kubernetes/scheme",
-    "kubernetes/typed/admissionregistration/v1alpha1",
     "kubernetes/typed/admissionregistration/v1beta1",
     "kubernetes/typed/apps/v1",
     "kubernetes/typed/apps/v1beta1",
     "kubernetes/typed/apps/v1beta2",
+    "kubernetes/typed/auditregistration/v1alpha1",
     "kubernetes/typed/authentication/v1",
     "kubernetes/typed/authentication/v1beta1",
     "kubernetes/typed/authorization/v1",
     "kubernetes/typed/authorization/v1beta1",
     "kubernetes/typed/autoscaling/v1",
     "kubernetes/typed/autoscaling/v2beta1",
+    "kubernetes/typed/autoscaling/v2beta2",
     "kubernetes/typed/batch/v1",
     "kubernetes/typed/batch/v1beta1",
     "kubernetes/typed/batch/v2alpha1",
     "kubernetes/typed/certificates/v1beta1",
+    "kubernetes/typed/coordination/v1",
+    "kubernetes/typed/coordination/v1beta1",
     "kubernetes/typed/core/v1",
     "kubernetes/typed/events/v1beta1",
     "kubernetes/typed/extensions/v1beta1",
     "kubernetes/typed/networking/v1",
+    "kubernetes/typed/networking/v1beta1",
+    "kubernetes/typed/node/v1alpha1",
+    "kubernetes/typed/node/v1beta1",
     "kubernetes/typed/policy/v1beta1",
     "kubernetes/typed/rbac/v1",
     "kubernetes/typed/rbac/v1alpha1",
     "kubernetes/typed/rbac/v1beta1",
+    "kubernetes/typed/scheduling/v1",
     "kubernetes/typed/scheduling/v1alpha1",
+    "kubernetes/typed/scheduling/v1beta1",
     "kubernetes/typed/settings/v1alpha1",
     "kubernetes/typed/storage/v1",
     "kubernetes/typed/storage/v1alpha1",
     "kubernetes/typed/storage/v1beta1",
     "pkg/apis/clientauthentication",
     "pkg/apis/clientauthentication/v1alpha1",
+    "pkg/apis/clientauthentication/v1beta1",
     "pkg/version",
     "plugin/pkg/client/auth/exec",
     "plugin/pkg/client/auth/oidc",
@@ -632,35 +670,75 @@
     "tools/metrics",
     "tools/pager",
     "tools/record",
+    "tools/record/util",
     "tools/reference",
     "transport",
-    "util/buffer",
     "util/cert",
+    "util/connrotation",
     "util/flowcontrol",
     "util/homedir",
-    "util/integer",
+    "util/keyutil",
     "util/retry",
   ]
   pruneopts = "T"
-  revision = "23781f4d6632d88e869066eaebb743857aa1ef9b"
-  version = "v7.0.0"
+  revision = "6ee68ca5fd8355d024d02f9db0b3b667e8357a0f"
+  version = "v11.0.0"
 
 [[projects]]
-  digest = "1:48edb837aa6718415bc86de07fa5a6f8eab31bb8155262b27ebe00e763f25efb"
+  digest = "1:2d821667dbd520a7ef31bdc923543f197ba30021b4317fd8871618ada52f23b0"
   name = "k8s.io/code-generator"
-  packages = ["cmd/client-gen/types"]
+  packages = [
+    "cmd/client-gen",
+    "cmd/client-gen/args",
+    "cmd/client-gen/generators",
+    "cmd/client-gen/generators/fake",
+    "cmd/client-gen/generators/scheme",
+    "cmd/client-gen/generators/util",
+    "cmd/client-gen/path",
+    "cmd/client-gen/types",
+    "cmd/conversion-gen",
+    "cmd/conversion-gen/args",
+    "cmd/conversion-gen/generators",
+    "cmd/deepcopy-gen",
+    "cmd/deepcopy-gen/args",
+    "cmd/defaulter-gen",
+    "cmd/defaulter-gen/args",
+    "cmd/informer-gen",
+    "cmd/informer-gen/args",
+    "cmd/informer-gen/generators",
+    "cmd/lister-gen",
+    "cmd/lister-gen/args",
+    "cmd/lister-gen/generators",
+    "pkg/namer",
+    "pkg/util",
+  ]
   pruneopts = "T"
-  revision = "c7b82f7132f7071513e34dff19f450c96293a745"
+  revision = "50b561225d70b3eb79a1faafd3dfe7b1a62cbe73"
+  version = "kubernetes-1.14.1"
 
 [[projects]]
   digest = "1:f90b6152db4c5ee920eda292be42f4103498429d4238ea3e4003972b0dc5cfdd"
   name = "k8s.io/gengo"
   packages = [
+    "args",
+    "examples/deepcopy-gen/generators",
+    "examples/defaulter-gen/generators",
+    "examples/set-gen/sets",
+    "generator",
     "namer",
+    "parser",
     "types",
   ]
   pruneopts = "T"
   revision = "7338e4bfd6915369a1375890db1bbda0158c9863"
+
+[[projects]]
+  digest = "1:8120e234193b48135028c8fe897443ecdb5ffd70065019dec34884ba0bc108ad"
+  name = "k8s.io/klog"
+  packages = ["."]
+  pruneopts = "T"
+  revision = "e531227889390a39d9533dde61f590fe9f4b0035"
+  version = "v0.3.0"
 
 [[projects]]
   branch = "master"
@@ -670,6 +748,26 @@
   pruneopts = "T"
   revision = "0317810137be915b9cf888946c6e115c1bfac693"
 
+[[projects]]
+  branch = "master"
+  digest = "1:42a4fb910742f6aa40f27b38fafde8fc49bb12ee6a056df7df239d99527a46d4"
+  name = "k8s.io/utils"
+  packages = [
+    "buffer",
+    "integer",
+    "trace",
+  ]
+  pruneopts = "T"
+  revision = "8fab8cb257d50c8cf94ec9771e74826edbb68fb5"
+
+[[projects]]
+  digest = "1:7719608fe0b52a4ece56c2dde37bedd95b938677d1ab0f84b8a7852e4c59f849"
+  name = "sigs.k8s.io/yaml"
+  packages = ["."]
+  pruneopts = "T"
+  revision = "fd68e9863619f6ec2fdd8625fe1f02e7c877e480"
+  version = "v1.1.0"
+
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
@@ -677,6 +775,7 @@
     "github.com/PaesslerAG/jsonpath",
     "github.com/onsi/ginkgo",
     "github.com/onsi/ginkgo/config",
+    "github.com/onsi/ginkgo/ginkgo",
     "github.com/onsi/ginkgo/reporters",
     "github.com/onsi/gomega",
     "github.com/onsi/gomega/types",
@@ -717,7 +816,13 @@
     "k8s.io/client-go/tools/clientcmd",
     "k8s.io/client-go/tools/record",
     "k8s.io/client-go/util/flowcontrol",
+    "k8s.io/code-generator/cmd/client-gen",
     "k8s.io/code-generator/cmd/client-gen/types",
+    "k8s.io/code-generator/cmd/conversion-gen",
+    "k8s.io/code-generator/cmd/deepcopy-gen",
+    "k8s.io/code-generator/cmd/defaulter-gen",
+    "k8s.io/code-generator/cmd/informer-gen",
+    "k8s.io/code-generator/cmd/lister-gen",
   ]
   solver-name = "gps-cdcl"
   solver-version = 1

--- a/cassandra-operator/Gopkg.toml
+++ b/cassandra-operator/Gopkg.toml
@@ -1,28 +1,12 @@
-# Gopkg.toml example
-#
-# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
-# for detailed Gopkg.toml documentation.
-#
-# required = ["github.com/user/thing/cmd/thing"]
-# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
-#
-# [[constraint]]
-#   name = "github.com/user/project"
-#   version = "1.0.0"
-#
-# [[constraint]]
-#   name = "github.com/user/project2"
-#   branch = "dev"
-#   source = "github.com/myfork/project2"
-#
-# [[override]]
-#   name = "github.com/x/y"
-#   version = "2.4.0"
-#
-# [prune]
-#   non-go = false
-#   go-tests = true
-#   unused-packages = true
+required = [
+  "k8s.io/code-generator/cmd/client-gen",
+  "k8s.io/code-generator/cmd/conversion-gen",
+  "k8s.io/code-generator/cmd/deepcopy-gen",
+  "k8s.io/code-generator/cmd/defaulter-gen",
+  "k8s.io/code-generator/cmd/informer-gen",
+  "k8s.io/code-generator/cmd/lister-gen",
+  "github.com/onsi/ginkgo/ginkgo",
+]
 
 [[constraint]]
   name = "github.com/sirupsen/logrus"
@@ -30,7 +14,7 @@
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "v7.0.0"
+  version = "v11.0.0"
 
 [[constraint]]
   name = "github.com/onsi/ginkgo"
@@ -49,19 +33,12 @@
   version = "0.0.3"
 
 [[constraint]]
-  name = "k8s.io/code-generator"
-  revision = "c7b82f7132f7071513e34dff19f450c96293a745"
-
-[[override]]
-  # fixed to this version in order to avoid compilation error.
   name = "k8s.io/apimachinery"
-  revision = "800fcb029b488923f14852306f0c98c11d0f360c"
+  version = "kubernetes-1.14.1"
 
-[[override]]
-  # fixed to this revision to avoid picking up a change in the logging framework used by gengo, which causes issues with
-  # our current set of kubernetes dependencies.
-  name = "k8s.io/gengo"
-  revision = "7338e4bfd6915369a1375890db1bbda0158c9863"
+[[constraint]]
+  name = "k8s.io/code-generator"
+  version = "kubernetes-1.14.1"
 
 [[override]]
   # required to work around a bug in dep: https://github.com/golang/dep/issues/1799

--- a/cassandra-operator/Makefile
+++ b/cassandra-operator/Makefile
@@ -121,7 +121,7 @@ validate-licences:
 	set -e ;\
 	restricted=$$(paste -s -d ',' restricted-licences.txt) ;\
 	projects=$$(dep status -f='vendor/{{ .ProjectRoot }} ') ;\
-	licence-compliance-checker -L error -E -r $$restricted -o vendor/github.com/ghodss/yaml=BSD-3-Clause -o vendor/github.com/golang/protobuf=BSD-3-Clause -o vendor/github.com/gogo/protobuf=BSD-3-Clause -o vendor/github.com/hashicorp/golang-lru=MPL-2.0 $$projects ;\
+	licence-compliance-checker -L error -E -r $$restricted -o vendor/sigs.k8s.io/yaml=BSD-3-Clause -o vendor/github.com/golang/protobuf=BSD-3-Clause -o vendor/github.com/gogo/protobuf=BSD-3-Clause -o vendor/github.com/hashicorp/golang-lru=MPL-2.0 $$projects ;\
 
 clean: common-docker-clean
 	@echo "== clean"

--- a/cassandra-operator/pkg/client/clientset/versioned/clientset.go
+++ b/cassandra-operator/pkg/client/clientset/versioned/clientset.go
@@ -12,8 +12,6 @@ import (
 type Interface interface {
 	Discovery() discovery.DiscoveryInterface
 	CoreV1alpha1() corev1alpha1.CoreV1alpha1Interface
-	// Deprecated: please explicitly pick a version if possible.
-	Core() corev1alpha1.CoreV1alpha1Interface
 }
 
 // Clientset contains the clients for groups. Each group has exactly one
@@ -25,12 +23,6 @@ type Clientset struct {
 
 // CoreV1alpha1 retrieves the CoreV1alpha1Client
 func (c *Clientset) CoreV1alpha1() corev1alpha1.CoreV1alpha1Interface {
-	return c.coreV1alpha1
-}
-
-// Deprecated: Core retrieves the default version of CoreClient.
-// Please explicitly pick a version.
-func (c *Clientset) Core() corev1alpha1.CoreV1alpha1Interface {
 	return c.coreV1alpha1
 }
 

--- a/cassandra-operator/pkg/client/clientset/versioned/fake/clientset_generated.go
+++ b/cassandra-operator/pkg/client/clientset/versioned/fake/clientset_generated.go
@@ -59,8 +59,3 @@ var _ clientset.Interface = &Clientset{}
 func (c *Clientset) CoreV1alpha1() corev1alpha1.CoreV1alpha1Interface {
 	return &fakecorev1alpha1.FakeCoreV1alpha1{Fake: &c.Fake}
 }
-
-// Core retrieves the CoreV1alpha1Client
-func (c *Clientset) Core() corev1alpha1.CoreV1alpha1Interface {
-	return &fakecorev1alpha1.FakeCoreV1alpha1{Fake: &c.Fake}
-}

--- a/cassandra-operator/pkg/client/clientset/versioned/scheme/register.go
+++ b/cassandra-operator/pkg/client/clientset/versioned/scheme/register.go
@@ -8,6 +8,7 @@ import (
 	runtime "k8s.io/apimachinery/pkg/runtime"
 	schema "k8s.io/apimachinery/pkg/runtime/schema"
 	serializer "k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 )
 
 var Scheme = runtime.NewScheme()
@@ -35,5 +36,5 @@ var AddToScheme = localSchemeBuilder.AddToScheme
 
 func init() {
 	v1.AddToGroupVersion(Scheme, schema.GroupVersion{Version: "v1"})
-	must(AddToScheme(Scheme))
+	utilruntime.Must(AddToScheme(Scheme))
 }

--- a/cassandra-operator/pkg/client/clientset/versioned/typed/cassandra/v1alpha1/cassandra.go
+++ b/cassandra-operator/pkg/client/clientset/versioned/typed/cassandra/v1alpha1/cassandra.go
@@ -3,6 +3,8 @@
 package v1alpha1
 
 import (
+	"time"
+
 	v1alpha1 "github.com/sky-uk/cassandra-operator/cassandra-operator/pkg/apis/cassandra/v1alpha1"
 	scheme "github.com/sky-uk/cassandra-operator/cassandra-operator/pkg/client/clientset/versioned/scheme"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
@@ -60,11 +62,16 @@ func (c *cassandras) Get(name string, options v1.GetOptions) (result *v1alpha1.C
 
 // List takes label and field selectors, and returns the list of Cassandras that match those selectors.
 func (c *cassandras) List(opts v1.ListOptions) (result *v1alpha1.CassandraList, err error) {
+	var timeout time.Duration
+	if opts.TimeoutSeconds != nil {
+		timeout = time.Duration(*opts.TimeoutSeconds) * time.Second
+	}
 	result = &v1alpha1.CassandraList{}
 	err = c.client.Get().
 		Namespace(c.ns).
 		Resource("cassandras").
 		VersionedParams(&opts, scheme.ParameterCodec).
+		Timeout(timeout).
 		Do().
 		Into(result)
 	return
@@ -72,11 +79,16 @@ func (c *cassandras) List(opts v1.ListOptions) (result *v1alpha1.CassandraList, 
 
 // Watch returns a watch.Interface that watches the requested cassandras.
 func (c *cassandras) Watch(opts v1.ListOptions) (watch.Interface, error) {
+	var timeout time.Duration
+	if opts.TimeoutSeconds != nil {
+		timeout = time.Duration(*opts.TimeoutSeconds) * time.Second
+	}
 	opts.Watch = true
 	return c.client.Get().
 		Namespace(c.ns).
 		Resource("cassandras").
 		VersionedParams(&opts, scheme.ParameterCodec).
+		Timeout(timeout).
 		Watch()
 }
 
@@ -134,10 +146,15 @@ func (c *cassandras) Delete(name string, options *v1.DeleteOptions) error {
 
 // DeleteCollection deletes a collection of objects.
 func (c *cassandras) DeleteCollection(options *v1.DeleteOptions, listOptions v1.ListOptions) error {
+	var timeout time.Duration
+	if listOptions.TimeoutSeconds != nil {
+		timeout = time.Duration(*listOptions.TimeoutSeconds) * time.Second
+	}
 	return c.client.Delete().
 		Namespace(c.ns).
 		Resource("cassandras").
 		VersionedParams(&listOptions, scheme.ParameterCodec).
+		Timeout(timeout).
 		Body(options).
 		Do().
 		Error()

--- a/cassandra-operator/pkg/client/clientset/versioned/typed/cassandra/v1alpha1/fake/fake_cassandra.go
+++ b/cassandra-operator/pkg/client/clientset/versioned/typed/cassandra/v1alpha1/fake/fake_cassandra.go
@@ -115,7 +115,7 @@ func (c *FakeCassandras) DeleteCollection(options *v1.DeleteOptions, listOptions
 // Patch applies the patch and returns the patched cassandra.
 func (c *FakeCassandras) Patch(name string, pt types.PatchType, data []byte, subresources ...string) (result *v1alpha1.Cassandra, err error) {
 	obj, err := c.Fake.
-		Invokes(testing.NewPatchSubresourceAction(cassandrasResource, c.ns, name, data, subresources...), &v1alpha1.Cassandra{})
+		Invokes(testing.NewPatchSubresourceAction(cassandrasResource, c.ns, name, pt, data, subresources...), &v1alpha1.Cassandra{})
 
 	if obj == nil {
 		return nil, err

--- a/cassandra-operator/pkg/client/informers/externalversions/internalinterfaces/factory_interfaces.go
+++ b/cassandra-operator/pkg/client/informers/externalversions/internalinterfaces/factory_interfaces.go
@@ -11,6 +11,7 @@ import (
 	cache "k8s.io/client-go/tools/cache"
 )
 
+// NewInformerFunc takes versioned.Interface and time.Duration to return a SharedIndexInformer.
 type NewInformerFunc func(versioned.Interface, time.Duration) cache.SharedIndexInformer
 
 // SharedInformerFactory a small interface to allow for adding an informer without an import cycle
@@ -19,4 +20,5 @@ type SharedInformerFactory interface {
 	InformerFor(obj runtime.Object, newFunc NewInformerFunc) cache.SharedIndexInformer
 }
 
+// TweakListOptionsFunc is a function that transforms a v1.ListOptions.
 type TweakListOptionsFunc func(*v1.ListOptions)

--- a/cassandra-operator/pkg/operator/operations/operations_test.go
+++ b/cassandra-operator/pkg/operator/operations/operations_test.go
@@ -411,3 +411,5 @@ func (r *stubEventRecorder) Eventf(object runtime.Object, eventtype, reason, mes
 }
 func (r *stubEventRecorder) PastEventf(object runtime.Object, timestamp metav1.Time, eventtype, reason, messageFmt string, args ...interface{}) {
 }
+func (r *stubEventRecorder) AnnotatedEventf(object runtime.Object, annotations map[string]string, eventtype, reason, messageFmt string, args ...interface{}) {
+}

